backend_new/test/integration/listing.e2e-spec.ts

```diff
@@ -171,4 +171,28 @@ describe('Listing Controller Tests', () => {
     expect(res.body.items.length).toEqual(1);
     expect(res.body.items[0].name).toEqual('name: 51');
   });
+
+  it('retrieveListings test', async () => {
```

Comment:
Would love to see more descriptive test names. Using "it" as part of the test name I find to be helpful, like "should do x"

Reply:
nice call out, I updated the descriptions lemme know if those are giving the right vibe

Reply:
@YazeedLoonat , following up on this one, I agree that I like the "should do x" suggestion. Your updates are an improvement and can you do another iteration. For example:
`testing list endpoint without params` => `should retrieve list without params`